lib/Service/LoginService.php

```diff
@@ -253,6 +260,22 @@ public function login($profile, $userSession, $request)
                 }
             }
 
+            if ($this->config->getSystemValue('oidc_login_update_avatar', false) && $profile[$attr['photoURL']]) {
```

Comment:
According to the standard, the `picture` claim is not mandatory in the IdPs response (see [UserInfo Response](https://openid.net/specs/openid-connect-core-1_0.html#UserInfoResponse) as an example).

Currently, not having the corresponding claim in the `profile` array results in `Error: Undefined array key "picture" at /var/www/html/custom_apps/oidc_login/lib/Service/LoginService.php#263`. So we should rather check whether the `$profile` array has the `$attr['photoURL']` key. Alternatively, as we now have the feature flag, we could log a warning if it is enabled but the claim could not be found in the profile.